### DISCUSSION TITLE
Licence restrictions

### DISCUSSION THREAD

**Discussion by anthropoy** [NVIDIA/warp#7] [General]
Awesome work! But when I read the licence details, it says it's restricted to non-commercial use only...can that be reviewed? It's a shame that such a powerful tool is restricted in such a way.

**Comment by mmacklin** [comment]
Thanks for the feedback, I will raise your concerns with the legal team here for review.

**Comment by ib00** [comment]
Has there been any change in the licensing of "warp"? Is it still restricted to non-commercial use?

**Comment by mehdiataei** [comment]
@mmacklin We also support this change. We would like to use this in combination with (https://github.com/Autodesk/XLB) which is also a collaborative effort with Nvidia. However, the license limitations prohibits us from doing so as it is incompatible with Apache 2.0.

**Comment by edwilcox1** [comment]
The announcement was made on the warp discord channel, but in case anybody looking here didn't see it, the license has been updated to allow for commercial use now.

**Comment by mmacklin** [comment]
Another update here, we have officially re-licensed to Apache v2.0.
